examples/ampersand/.gitignore

```diff
@@ -0,0 +1,13 @@
+node_modules/.bin/*
+node_modules/browserify/*
+node_modules/watchify/*
+node_modules/jadeify/*
+node_modules/jade/*
+node_modules/debounce/*
+node_modules/ampersand-*/*
```

Comment:
I think you can drop the end star on these as ignoring the folder itself will work the same.